src/CarwowTheme/Filters.elm

```diff
@@ -1,27 +1,28 @@
-module CarwowTheme.Filters exposing (select, filterView, standardFilterView, FilterGroupItem, filterGroup)
+module CarwowTheme.Filters exposing (select, filterView, standardFilterView, FilterGroupItem, filterGroup, filterGroupWithExpander)
 
 {-| Filters
 
 
 # Exports
 
-@docs select, filterView, standardFilterView, FilterGroupItem, filterGroup
+@docs select, filterView, standardFilterView, FilterGroupItem, filterGroup, filterGroupWithExpander
 
 -}
 
 import CarwowTheme.Icons exposing (icon)
 import CarwowTheme.Inputs exposing (select)
-import Html exposing (div, span, text, ul, li)
-import Html.Attributes exposing (class)
+import Html exposing (div, span, text, ul, li, a)
+import Html.Attributes exposing (class, attribute, href, id, property)
 import Html.Events exposing (onCheck)
-
+import Json.Encode
 
 {-| Placeholder
 -}
 type alias FilterGroupItem msg =
     { filterId : String
     , filterLabel : String
     , filterValue : Bool
+    , filterDescription : String
```

Comment:
`Maybe String` - this way the type encapsulates whether or not there's a description.